src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
+
+import java.util.ArrayList;
+
+import duke.maincomponents.TaskList;
+import duke.exceptions.DukeException;
+import duke.maincomponents.Storage;
+import duke.maincomponents.Ui;
+import duke.task.Task;
+
+/**
+ * Deadline Command, which adds a deadline task to Duke's TaskList when executed
+ */
+public class DeadlineCommand implements Command {
+    private ArrayList<String> eventDescription;
+
+    /**
+     * Constuct a DeadlineCommand object
+     * @param a Arraylist where index 0 contains the description of the deadline and index 1
```

Comment:
Perhaps you could include an empty line between description and parameter section to comply with the coding standards?